### ISSUE TITLE
SassError: Index out of range

### ISSUE THREAD

**Issue by JoomFX** [telerik/kendo-themes#4556]
**Describe the bug**
There are several users experiencing this issue after upgrading from an older Kendo Themes version:

```
./src/styles.scss - Error: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Index out of range
   ╷
23 │     @return function-exists( $name );
   │             ^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  node_modules\@progress\kendo-theme-core\scss\functions\_meta.import.scss 23:13     k-meta-function-exists()
  node_modules\@progress\kendo-theme-utils\scss\_mixins.scss 4:19                    generate-utils()
  node_modules\@progress\kendo-theme-utils\scss\layout\_aspect-ratio.scss 29:5       kendo-utils--layout--aspect-ratio()
  node_modules\@progress\kendo-theme-utils\scss\layout\index.import.scss 17:5        kendo-utils--layout()
  node_modules\@progress\kendo-theme-utils\scss\index.import.scss 25:5               kendo-utils()
  node_modules\@progress\kendo-theme-default\scss\utils\_index.scss 26:9             @content
  node_modules\@progress\kendo-theme-core\scss\module-system\index.import.scss 35:9  module-render()
  node_modules\@progress\kendo-theme-default\scss\utils\_index.scss 25:5             kendo-utils--styles()
  node_modules\@progress\kendo-theme-default\scss\utils\_index.scss 31:5             @import
  node_modules\@progress\kendo-theme-default\scss\icons\_index.scss 2:9              @import
  src\styles.scss 4:9                                                                root stylesheet
``` 

Some of the users were able to work around the issue by changing the order of their `@import`'s, but it is a bug in Kendo Themes that was introduced in v6.2.0.

Here is a list of some of the users (tickets and issues) that experience this issue:
https://github.com/telerik/kendo-themes/issues/4231
ticket 1612296
ticket 1612746
ticket 1606635
ticket 1605984

**To reproduce**
I was able to reproduce the issue by importing Angular Material Theming before Kendo Theme Default in the **styles.scss** file of a brand new Angular application (most probably there are other `@import`'s causing the issue, it is not limited to Angular Material Theming only):

```
@import '@angular/material/theming';
@import "@progress/kendo-theme-default/dist/all.scss";
```

If you change the order of the imports (if you import Kendo Theme Default first) the issue is not present.

**Also, if you downgrade the Kendo Themes to version 6.1.0 the issue will be gone.**

1. Download the attached Angular 14 app.
1. Run `npm i`
1. Run `ng s`

[angular14.zip](https://github.com/telerik/kendo-themes/files/11768585/angular14.zip)

**Expected behavior**
Angular (and not only) applications should compile without errors, no matter what the order of the `@import`'s is.

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery
* Kendo UI for Angular
* Kendo UI for React
* Kendo UI for Vue
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* All

**Comment by bedwards-cms** [comment]
Downgrading to v6.1.0 For Kendo/React did not resolve the issue for me.  Then I tried version 6.4 and still get this error.

**Comment by Juveniel** [comment]
@JoomFX could you take a look please ?

**Comment by JoomFX** [comment]
@bedwards-cms Could you please send us a sample app where we can see the issue and where we can see what other libraries are being imported, etc?
We fixed this issue in v6.4 but there might be another thing that also needs to be fixed. We just need to be able to reproduce the issue so we can investigate it further.

**Comment by bedwards-cms** [comment]
So I looked at the commit change for the other fix, and hardcoded it temporarily and saw it resolved the issue.   I then took it back out and did another npm install. It seems to be working now.

**Comment by bedwards-cms** [comment]
@JoomFX - ok it wasn't until I got version 6.6 that my issue was resolved.    This was what fixed it:
https://github.com/telerik/kendo-themes/pull/4557/files

﻿

**Comment by JoomFX** [comment]
@bedwards-cms Great! I'm glad the issue is resolved.
In my previous comment I said that we fixed this issue in v6.4.0 but we actually fixed it in [v6.5.0](https://github.com/telerik/kendo-themes/releases/tag/v6.5.0). Sorry about that.
